schema-guide.md

```diff
@@ -885,6 +885,7 @@ authors:
     references:
       - conference:
           country: DE
+          name: The conference name
```

Comment:
Adding `name` here was the simplest workaround, not sure if we will run into this (i.e. required properties of entities) more in the future.